opal/mca/rcache/vma/rcache_vma_tree.c

```diff
@@ -276,6 +277,23 @@ void mca_rcache_vma_tree_finalize(mca_rcache_vma_module_t* rcache)
                       mca_rcache_vma_tree_node_compare);
     OBJ_DESTRUCT(&rcache->vma_list);
     OBJ_DESTRUCT(&rcache->rb_tree);
+    OPAL_LIST_DESTRUCT(&rcache->vma_gc_list);
+}
+
+/**
+ * Clean the vma garbage collection list
+ *
+ * This function releases any deleted vma structures. It MUST be called
+ * with the VMA lock help and MAY NOT be called from a function that can
+ * be called by a memory hook.
+ */
+static void mca_rcache_vma_gc_clean (mca_rcache_vma_module_t* rcache)
+{
+    opal_list_item_t *item;
+
+    while (NULL != (item = opal_list_remove_first (&rcache->vma_gc_list))) {
+        OBJ_RELEASE(item);
+    }
```

Comment:
Any reason not to use OPAL_LIST_DESTRUCT here?

Or is it just to-MAY-to vs. to-MAH-to?